stress/common/src/main/java/alluxio/stress/common/FileSystemParameters.java

```diff
@@ -55,4 +61,19 @@ public ReadType convert(String value) {
       return ReadType.fromString(value);
     }
   }
+
+  /**
+   * @return FileSystemWriteType of this bench
+   * Converts from String to a valid FileSystemWriteType String.
+   */
+  public static class FileSystemParametersWriteTypeConverter implements IStringConverter<String> {
+    @Override
+    public String convert(String value) {
+      if (value.equals("MUST_CACHE")  || value.equals("CACHE_THROUGH")  || value.equals("THROUGH")
```

Comment:
Can you reuse `alluxio.client.WriteType`, checking if the string is one of the enum?